experimental/dds/tree2/src/test/shared-tree-core/branch.spec.ts

```diff
@@ -439,6 +435,54 @@ describe("Branches", () => {
 		assert.equal(branch.getHead().revision, nullRevisionTag);
 	});
 
+	describe("all nested forks and transactions are disposed and aborted when transaction is", () => {
+		const setUpNestedForks = (rootBranch: DefaultBranch) => {
+			change(rootBranch);
+			const fork1 = rootBranch.fork();
+			change(rootBranch);
+			rootBranch.startTransaction();
+			const fork2 = rootBranch.fork();
+			change(rootBranch);
+			const fork3 = rootBranch.fork();
+			change(fork3);
+			fork3.startTransaction();
+			change(fork3);
+			const fork4 = fork3.fork();
+
+			return {
+				disposedFroks: [fork2, fork3, fork4],
```

Comment:
```suggestion
				disposedForks: [fork2, fork3, fork4],
```

Reply:
The suggested change is the same as it is.

Reply:
The change is fixing the spelling error: disposedFroks => disposedForks